Ex_JDBC01/src/main/java/lt/verbus/repository/ExecutorRepository.java

```diff
@@ -0,0 +1,87 @@
+package lt.verbus.repository;
+
+import lt.verbus.config.QueriesMySql;
+import lt.verbus.model.Executor;
+import lt.verbus.model.Project;
+
+import java.sql.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExecutorRepository implements CrudRepository<Integer, Executor> {
```

Comment:
Speju cia bandai viena 'globalu' updateri? Bet matai, tu vistiek esi prisirises prie enticio, ir juos esi hardcodines (pvz "table.getString("qualification")"). Gal galim padaryti kazka universalaus? Pasiziurek i tai:

Class<?> clazz
clazz.getDeclaredFields() <-- Sitas tau grazintu kolekcija fieldu, is kuriu galetum pasiimti fieldName (pvz qualification)